data/topics/new.md

```diff
@@ -0,0 +1,65 @@
+---
+title: new
+keywords:
+  - memory
+  - leak
+  - object
+---
+
+In the PJS environment, code is transformed such that any object created in a typical fashion (using the `new` keyword or literal syntax) is stored in an array in memory, allowing the memory usage of programs to balloon in size if objects are dereferenced willy-nilly. The garbage collector cannot reclaim the memory, and programs will slow down as the array grows.
+
+Throughout KA's history, there have arisen several ways to deal with the problem. Aside from creating and using complex object pools that shift the burden onto the programmer, the following are methods of restoring "expected" behavior back to `new`
+
+1. Using Object.create (Commonly attributed to ([Bob Lyon](https://www.khanacademy.org/profile/kaid_1116520053705160540512971/projects) and [KWC](https://www.khanacademy.org/profile/kaid_1116520053705160540512971/projects))?)
+    ```js
+    (function NewFix () {
```

Comment:
Why are these IIFEs named?

Reply:
I tend to do that. You can remove it if you want, but it's a little more self-documenting than a comment

Reply:
I don't think there's an easy way for me to edit the code on your PR so we'll keep it as is. Maybe I'll edit it later down the road for uniformity.